src/screens/Home/useHomeController.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {useState} from 'react';
+import {Alert} from 'react-native';
+
+import {fetchEverything} from '~/services/resources/everything';
+import {ArticleJSON} from '~/@types/dtos/everythingDTO';
+
+const useHomeController = () => {
+  const [news, setNews] = useState<ArticleJSON[]>([]);
+
+  const fetchNews = React.useCallback(async () => {
```

Comment:
```suggestion
  const fetchNews = useCallback(async () => {
```

Import it on the React import statement